types/permit/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-void-expression": false
```

Comment:
Where is this needed? Can you get rid of it?

Reply:
Here:
```javascript
export class Permit {
     constructor(options: PermitOptions);
     check(req: IncomingMessage): void;
     fail(res: ServerResponse): void;
 }
```

The Permit class has these functions but they throw and error instead of returning anything. I
wasn't sure how to handle that. But since there is nothing to return I added void as return type, which resulted in the tslint error, so I added the rule.